tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_kra.yml

```diff
@@ -1,3 +1,6 @@
+- name: Sleep for a while to give time of any other instances to come up.
+  shell: sleep 7s
```

Comment:
Nice hack :) Should we wait for `getStatus` to return a value instead, perhaps? Just a thought though.

Reply:
@06shalini do you have thoughts?

Reply:
@cipherboy I can try that, I am in process of putting another PR for ldaps topology. Would cover this pointer there. I will have to play around ansible command for that.

Reply:
Sure, let's wait for another PR. We could also add that as a CLI endpoint maybe. Make it easier.